### DISCUSSION TITLE
What is the type of `duration`?

### DISCUSSION THREAD

**Discussion by TornaxO7** [elkowar/eww#568] [Q&A] [answered]
Hello! Currently I'm getting the following error messages:
```
5 │    :time-range 10
  │                ── `10` is not of type `duration`
```
But changing it into `10.0` doesn't work either:
```
5 │    :time-range 10.0
  │                ──── `10.0` is not of type `duration`
```
what should I enter in `duration` then?

**Comment by viandoxdev** [comment, marked as answer]
A time followed by an s (seconds) or maybe ls (milliseconds) in a string

**Reply by TornaxO7** [reply]
Thanks!
